consensus/ibft/ibft.go

```diff
@@ -360,21 +360,21 @@ func (i *Ibft) createKey() error {
 			// The validator key is present in the secrets manager, load it
 			validatorKey, readErr := crypto.ReadConsensusKey(i.secretsManager)
 			if readErr != nil {
-				return fmt.Errorf("unable to read validator key from Secrets Manager, %w", readErr)
+				return fmt.Errorf("unable to read validator key from SecretsConfigPath Manager, %w", readErr)
 			}
 
 			key = validatorKey
 		} else {
 			// The validator key is not present in the secrets manager, generate it
 			validatorKey, validatorKeyEncoded, genErr := crypto.GenerateAndEncodePrivateKey()
 			if genErr != nil {
-				return fmt.Errorf("unable to generate validator key for Secrets Manager, %w", genErr)
+				return fmt.Errorf("unable to generate validator key for SecretsConfigPath Manager, %w", genErr)
 			}
 
 			// Save the key to the secrets manager
 			saveErr := i.secretsManager.SetSecret(secrets.ValidatorKey, validatorKeyEncoded)
 			if saveErr != nil {
-				return fmt.Errorf("unable to save validator key to Secrets Manager, %w", saveErr)
+				return fmt.Errorf("unable to save validator key to SecretsConfigPath Manager, %w", saveErr)
```

Comment:
Typo in refactoring the error message

Reply:
Fixed 👍 

https://github.com/0xPolygon/polygon-edge/pull/410/commits/788abbc3667cd6c5f0d2f8b93e9302bf21606edd